homeassistant/components/squeezebox/media_player.py

```diff
@@ -240,6 +245,13 @@ def __init__(self, coordinator: SqueezeBoxPlayerUpdateCoordinator) -> None:
             model=player.model,
             manufacturer=_manufacturer,
         )
+        self._known_apps: set[str] = set()
+        self._squeezebox_maps = Squeezebox_maps(
+            content_type_media_class=CONTENT_TYPE_MEDIA_CLASS,
+            content_type_to_child_type=CONTENT_TYPE_TO_CHILD_TYPE,
+            squeezebox_id_by_type=SQUEEZEBOX_ID_BY_TYPE,
+            media_type_to_squeezebox=MEDIA_TYPE_TO_SQUEEZEBOX,
+        )
```

Comment:
You're assigning the instance, which means they will be modified later on. Do this instead:
1. Give the fields a default factory, so they are initialized to empty maps and don't have to be assigned in the constructor.
2. Implement `__post_init__` and copy the values from the constants into the instance variables.